Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+flask = "*"
+flask-restful = "*"
```

Comment:
Not sure what the feeling is, but I don't want us using a 3rd party tool to make the API when we could just use flask by itself.

Reply:
I've reworked the PR so that it doesn't use flask-restful anymore.